android/app/src/main/java/com/lndr/PayPalModule.java

```diff
@@ -72,13 +73,20 @@ public static String getConfigProperty(String key, Context context) {
     public void onActivityResult (Activity activity, int requestCode, int resultCode, Intent data) {
       if (resultCode == Activity.RESULT_OK) {
         PayPalAuthorization auth = data.getParcelableExtra(PayPalProfileSharingActivity.EXTRA_RESULT_AUTHORIZATION);
+        PaymentConfirmation confirm = data.getParcelableExtra(PaymentActivity.EXTRA_RESULT_CONFIRMATION);
         if (auth != null) {
           try {
             String authorization_code = auth.getAuthorizationCode();
             promise.resolve(authorization_code);
           } catch (Exception e) {
             promise.reject("Unknown error", e);
           }
+        } else if (confirm != null) {
+          try {
+            promise.resolve("Success");
```

Comment:
We should be passing the confirmation token/code here, not a human-readable string.